tasks/installubuntu.yml

```diff
@@ -0,0 +1,47 @@
+# roles/mariadb/tasks/install.yml
+---
+- name: Check Python3 present
+  stat:
+    path: /usr/bin/python3
+  register: python3ok
+
+- name: correct python version selected
```

Comment:
I fear this task could have side effects beyond the execution of this role. Selecting a system-wide Python version may affect other components of the system than the MariaDB installation.